src/Invoice/Domain/User/Profile.php

```diff
@@ -6,7 +6,7 @@
 
 use Invoice\Domain\VatNumber;
 
-final class Profile
+class Profile
```

Comment:
Tutaj zostawił bym `final` i inaczej ogarnął mapping

Reply:
jak możesz to zrób ten mapping